src/main/resources/changesets/changelog-consent-67.0.xml

```diff
@@ -0,0 +1,59 @@
+            </column>
+            <column name="era_commons_id" type="varchar(255)">
+            </column>
+            <column name="user_name" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="user_email" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="create_user_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="create_date" type="DATETIME">
+                <constraints nullable="false"/>
+            </column>
+            <column name="update_user_id" type="bigint"/>
+            <column name="update_date" type="DATETIME"/>
+        </createTable>
+        <addForeignKeyConstraint baseColumnNames="user_id" baseTableName="library_card" constraintName="fkLibraryCardUser"
+                                 deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION"
+                                 referencedColumnNames="dacuserid" referencedTableName="dacuser"
+                                 referencesUniqueColumn="false"/>
+        <addForeignKeyConstraint baseColumnNames="institution_id" baseTableName="library_card" constraintName="fkLibraryCardInstitution"
+                                 deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION"
+                                 referencedColumnNames="institution_id" referencedTableName="institution"
+                                 referencesUniqueColumn="false"/>
+        <addForeignKeyConstraint baseColumnNames="create_user_id" baseTableName="library_card" constraintName="fkLibraryCardCreateUser"
+                                 deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION"
+                                 referencedColumnNames="dacuserid" referencedTableName="dacuser"
+                                 referencesUniqueColumn="false"/>
+        <addForeignKeyConstraint baseColumnNames="update_user_id" baseTableName="library_card" constraintName="fkLibraryCardUpdateUser"
+                                 deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION"
+                                 referencedColumnNames="dacuserid" referencedTableName="dacuser"
+                                 referencesUniqueColumn="false"/>
+        <rollback>
+            <dropTable tableName="library_card"/>
+            <dropForeignKeyConstraint baseTableName="library_card" constraintName="fkLibraryCardUser"/>
+            <dropForeignKeyConstraint baseTableName="library_card" constraintName="fkLibraryCardInstitution"/>
+            <dropForeignKeyConstraint baseTableName="library_card" constraintName="fkLibraryCardCreateUser"/>
+            <dropForeignKeyConstraint baseTableName="library_card" constraintName="fkLibraryCardUpdateUser"/>
```

Comment:
Have not tested this, but I think you can just add `cascadeConstraints="true"` to `dropTable` instead of explicitly adding all of the constraints one at a time.